go.mod

```diff
@@ -100,9 +100,7 @@ require (
 	github.com/go-playground/validator/v10 v10.4.1 // indirect
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/gobuffalo/envy v1.7.0 // indirect
-	github.com/gobuffalo/logger v1.0.0 // indirect
```

Comment:
Don't forget to remove it from the dependency list at the top of go.mod too

Reply:
Done!

Reply:
You probably also need a `go mod tidy` now? There's still some references in `go.sum`

Reply:
This is odd, I still see `packr` in the `go.sum` even after I run a `go mod tidy`. Even after manually deleting those entries in `go.sum` and re-calling `go mod tidy` it still adds them back.

Reply:
Actually, another thing I noticed is that `go mod tidy` only removes some versions of a dependency from `go.sum` and not others. Not sure why that is. E.g `godirwalk`, `godotenv`, `go-internal`, and `logrus` still have entries in `go.sum`.

Reply:
So despite this being somewhat annoying, I think this is alright. The `go.sum` file basically just has hashes of modules at certain versions, and it does this to verify dependencies it downloaded (as a security feature). It doesn't mean that the dep _will_ be downloaded though, so it's not dependency bloat. The `go.mod` file however is where deps are declared and that's the source of truth on what modules are required by the app.